dev/tools/gen_defaults/data/snackbar.json

```diff
@@ -16,18 +16,18 @@
   "md.comp.snackbar.container.elevation": "md.sys.elevation.level3",
   "md.comp.snackbar.container.shadow-color": "shadow",
   "md.comp.snackbar.container.shape": "md.sys.shape.corner.extra-small",
-  "md.comp.snackbar.icon.color": "inverseOnSurface",
-  "md.comp.snackbar.icon.focus.icon.color": "inverseOnSurface",
-  "md.comp.snackbar.icon.focus.state-layer.color": "inverseOnSurface",
+  "md.comp.snackbar.icon.color": "onInverseSurface",
+  "md.comp.snackbar.icon.focus.icon.color": "onInverseSurface",
+  "md.comp.snackbar.icon.focus.state-layer.color": "onInverseSurface",
   "md.comp.snackbar.icon.focus.state-layer.opacity": "md.sys.state.focus.state-layer-opacity",
-  "md.comp.snackbar.icon.hover.icon.color": "inverseOnSurface",
-  "md.comp.snackbar.icon.hover.state-layer.color": "inverseOnSurface",
+  "md.comp.snackbar.icon.hover.icon.color": "onInverseSurface",
+  "md.comp.snackbar.icon.hover.state-layer.color": "onInverseSurface",
   "md.comp.snackbar.icon.hover.state-layer.opacity": "md.sys.state.hover.state-layer-opacity",
-  "md.comp.snackbar.icon.pressed.icon.color": "inverseOnSurface",
-  "md.comp.snackbar.icon.pressed.state-layer.color": "inverseOnSurface",
+  "md.comp.snackbar.icon.pressed.icon.color": "onInverseSurface",
+  "md.comp.snackbar.icon.pressed.state-layer.color": "onInverseSurface",
   "md.comp.snackbar.icon.pressed.state-layer.opacity": "md.sys.state.pressed.state-layer-opacity",
   "md.comp.snackbar.icon.size": 24.0,
-  "md.comp.snackbar.supporting-text.color": "inverseOnSurface",
+  "md.comp.snackbar.supporting-text.color": "onInverseSurface",
```

Comment:
I didn't realize these were out of sync. I will look into updating the token extractor script to fix it at that end.

Reply:
Great, thanks - didn't know there was a token script, so that's useful to know!